crates/core/tedge/src/cli/mod.rs

```diff
@@ -40,7 +40,14 @@ pub enum TEdgeOptMulticall {
         #[clap(subcommand)]
         cmd: TEdgeOpt,
 
-        #[clap(long = "config-dir", default_value = DEFAULT_TEDGE_CONFIG_PATH, global = true)]
+        // [env: TEDGE_CONFIG_DIR, default: /etc/tedge]
```

Comment:
You have to use a doc comment, otherwise `clap` doesn't use the comment for `--help`.

```suggestion
        /// [env: TEDGE_CONFIG_DIR, default: /etc/tedge]
```